pkg/agent/route/route_linux.go

```diff
@@ -642,6 +642,16 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet, localAntreaFl
 		}...)
 	}
 
+	// If AntreaIPAM is enabled, it SNATs the hostPort traffic which backend is AntreaIPAM VLAN Pod
```

Comment:
which backend is AntreaIPAM VLAN Pod -> to an AntreaIPAM VLAN Pod

Reply:
Could you add more information why we need SNAT in this case? Still not clear to me.

Reply:
Added detailed comments here.